src/ip_notifier.cpp

```diff
@@ -99,6 +98,80 @@ void CFDispatchRetain(dispatch_queue_t q) { dispatch_retain(q); }
 void CFDispatchRelease(dispatch_queue_t q) { dispatch_release(q); }
 using CFDispatchRef = CFRef<dispatch_queue_t, CFDispatchRetain, CFDispatchRelease>;
 
+#if TARGET_OS_IPHONE || defined TORRENT_FORCE_IOS_IP_NOTIFIER
```

Comment:
the way this is normally done is by having a define that controls the use of this feature, and then have ``config.hpp`` map platforms to sets of features to enable. This way you can still enable a feature on the command line. i.e. I think having a: ``TORRENT_USE_SC_NETWORK_REACHABILITY`` that enables this feature would be nicer than the ``TORRENT_FORCE_IOS_IP_NOTIFIER``